tests/pytests/unit/modules/test_mac_user.py

```diff
@@ -0,0 +1,423 @@
+"""
+    :codeauthor: Nicole Thomas <nicole@saltstack.com>
+"""
+
+import pytest
+import salt.modules.mac_user as mac_user
+from salt.exceptions import CommandExecutionError, SaltInvocationError
+from tests.support.mock import MagicMock, patch
+
+pwd = pytest.importorskip("pwd")
+
+
+@pytest.fixture
+def configure_loader_modules():
+    return {mac_user: {}}
+
+
+@pytest.fixture
+def mock_pwall():
+    return [
+        pwd.struct_passwd(
+            (
+                "_amavisd",
+                "*",
+                83,
+                83,
+                "AMaViS Daemon",
+                "/var/virusmails",
+                "/usr/bin/false",
+            )
+        ),
+        pwd.struct_passwd(
+            (
+                "_appleevents",
+                "*",
+                55,
+                55,
+                "AppleEvents Daemon",
+                "/var/empty",
+                "/usr/bin/false",
+            )
+        ),
+        pwd.struct_passwd(
+            (
+                "_appowner",
+                "*",
+                87,
+                87,
+                "Application Owner",
+                "/var/empty",
+                "/usr/bin/false",
+            )
+        ),
+    ]
+
+
+@pytest.fixture
+def mock_info_ret():
+    return {
+        "shell": "/bin/bash",
+        "name": "test",
+        "gid": 4376,
+        "groups": ["TEST_GROUP"],
+        "home": "/Users/foo",
+        "fullname": "TEST USER",
+        "uid": 4376,
+    }
+
+
+@pytest.mark.skipif(
```

Comment:
Can you create an issue since this one was closed (https://github.com/saltstack/salt/issues/10594) Looks like we are using a deprecated command (dscacheutil) as documented here (https://support.apple.com/en-us/HT202516) and we need to update it.

Reply:
Want me to remove that code?
 [we're not really using it anywhere](https://github.com/saltstack/salt/search?q=_flush_dscl_cache&type=)....

Reply:
On a follow-up PR?